Dockerfile

```diff
@@ -30,17 +36,21 @@ RUN ./check_plugins_location.sh v3 && \
 
 # Build registry, copying meta.yamls and index.json from builder
 FROM registry.centos.org/centos/httpd-24-centos7 AS registry
+USER 0
 COPY README.md .htaccess /var/www/html/
 COPY --from=builder /build/v3 /var/www/html/v3
+COPY ./scripts/*entrypoint.sh /usr/local/bin/
+
+WORKDIR /var/www/html
+ENTRYPOINT ["/usr/local/bin/uid_entrypoint.sh", "/usr/local/bin/entrypoint.sh"]
 
 # Offline build: cache .theia and .vsix files in registry itself and update metas
 FROM builder AS offline-builder
 
-# To only cache files from /latest/ folders, use ./cache_artifacts.sh v3 --latest-only 
-# and uncomment line above to remove files so they're not included in index.json -- RUN rm -fr $(find /build/v3 -name 'meta.yaml' | grep -v "/latest/" | grep -o ".*/")
 RUN ./cache_artifacts.sh v3 && chmod -R g+rwX /build
 
 # Offline registry: copy updated meta.yamls and cached extensions
 FROM registry AS offline-registry
+USER 0
```

Comment:
This root user. Are you doing for a particular reason or is it just a copy/paste to be consistent with RHEL image? Do you know why this is needed in RHEL image?

Reply:
we're already running as root, I'm just adding it there to be explicit. But it's not required.